src/main/webapp/app/markdown-editor/commands/bold.command.ts

```diff
@@ -15,36 +15,14 @@ export class BoldCommand extends Command {
     execute(): void {
         const selectedText = this.getSelectedText();
         let textToAdd = '';
-        let startSpace = '';
-        let endSpace = '';
-        let startIndx = 0;
-        let endIndx = 0;
-        const selectedTextLength = selectedText.length;
 
-        if (selectedText.includes('**')) {
+        if (selectedText.charAt(0) === '*' && selectedText.charAt(selectedText.length - 1) === '*') {
             textToAdd = selectedText.slice(2, -2);
             this.insertText(textToAdd);
         } else {
-            for (let i = 0; i < selectedTextLength; i++) {
-                if (selectedText.charAt(i) === ' ') {
-                    startSpace = startSpace + ' ';
-                } else {
-                    startIndx = i;
-                    break;
-                }
-            }
-
-            for (let j = selectedTextLength; j >= 0; j--) {
-                if (selectedText.charAt(j - 1) === ' ') {
-                    endSpace = endSpace + ' ';
-                } else {
-                    endIndx = j;
-                    break;
-                }
-            }
-            const newTxt = selectedText.slice(startIndx, endIndx);
-            textToAdd = `**${newTxt}**`;
-            this.insertText(startSpace + textToAdd + endSpace);
+            const refinedString = this.ignoreWhiteSpace(selectedText);
+            textToAdd = `**${refinedString}**`;
+            this.addRefinedTxt(textToAdd);
```

Comment:
Try not to use unnecessary shortenings as e.g. txt, there is only one letter difference and it makes the reader wonder (is this related to a windows txt file? :wink: )